tools/internal_ci/helper_scripts/prepare_build_windows.bat

```diff
@@ -62,15 +65,18 @@ set DOTNET_CLI_TELEMETRY_OPTOUT=true
 
 @rem Only install Python interpreters if we are running Python tests
 If "%PREPARE_BUILD_INSTALL_DEPS_PYTHON%" == "true" (
-    powershell -File tools\internal_ci\helper_scripts\install_python_interpreters.ps1 || goto :error
+  echo "!TIME!: invoking install_python_interpreters.ps1"
+  powershell -File tools\internal_ci\helper_scripts\install_python_interpreters.ps1 || goto :error
 )
 
 @rem Needed for uploading test results to bigquery
 python -m pip install google-api-python-client oauth2client six==1.16.0 || goto :error
 
 git submodule update --init || goto :error
 
+echo "!TIME!: prepare_build_windows.bat exiting"
 goto :EOF
 
 :error
+echo "!TIME!: prepare_build_windows.bat exiting"
```

Comment:
nit: Can we add extra words to say this is an abort due to error?

Reply:
done.